### PR TITLE
remove metrics

### PR DESCRIPTION
**Pull request by felipemello1**
oops, i merged https://github.com/meta-pytorch/torchforge/pull/606 before i pushed my latest changes

### CODE REVIEW

File: apps/grpo/main.py [hunk 1]
```diff
@@ -183,14 +183,19 @@ def simple_grpo_loss(
     loss = -(mean_policy_loss - beta * mean_kl)
 
     # Log metrics
+    # TODO: Better design - have loss function return all metrics as a dict,
```
Comment (allenwang28) [inline apps/grpo/main.py:186]: do we want to do this now or in a separate PR?

Comment (felipemello1) [reply apps/grpo/main.py:186]: separate PR. I am refactoring the loss today
